packages/styled-components/src/components/primitives/Form/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { TextArea } from '.';
+import { action } from '@storybook/addon-actions';
+import { boolean, text, select } from '@storybook/addon-knobs';
+
+export default {
+  title: 'Primitives/Form/TextArea',
+  component: TextArea
+};
+
+export const Shapes = () => (
+  <>
+    <h1>TextArea shapes</h1>
+    {Object.keys(TextArea.Shapes).map((shape: any) => (
+      <>
+        <TextArea shape={TextArea.Shapes[shape]} value={shape} />
```

Comment:
missing a key prop (we will have to do it for all stories)

Reply:
Added for TextArea story